mlserver/sys_tracing/stapsdt_stub.py

```diff
@@ -0,0 +1,31 @@
+class Provider(object):
```

Comment:
Are these stubs just used when the underlying library is not present?

Reply:
The stubs are used both when the underlying library is not present and for any tracepoints disabled via configuration.